src/controller.cpp

```diff
@@ -263,11 +263,6 @@ void Controller::handshakeTimeout() {
 
   emit handshakeFailed(hop.m_serverPublicKey);
 
-  if (m_nextStep != None) {
-    deactivate();
-    return;
```

Comment:
@oskirby @Gela Is it possible that we _do_ want to deactivate here (or even just return) if nextStep == `Disconnect`, `Quit`, or `Update` ? (https://github.com/mozilla-mobile/mozilla-vpn-client/blob/7bc6f48b29647ee8179cca10a30413cee05be454/src/controller.h#L169C1-L176C5)